src/adapters/README.md

```diff
@@ -0,0 +1,430 @@
+1. Test your connection to MongoDB at `http://localhost:3001/ping`. <br /><br /> ![](https://gist.githubusercontent.com/sonyl-ciq/6fa9eff051e53e3f4337134788866cd2/raw/6bdbf9d4a7d4bf47a54b7c81376f7f029218f93a/ping.png)
+1. If you do not see `{"status":200}`, check that MongoDB is running and accepting connections.
+
+We're ready to construct our storage adapter.
+
+## The Storage Adapter
+
+Your use case for a storage adapter may be highly nuanced, so we'll begin with only the essentials of our storage adapter. At its core, there are four methods needed: **get**, **save**, **keys**, and **delete**.
+
+We'll begin our adapter in _src/adapters/MongoStorageAdapter.js_ as such:
+
+```js
+const BaseStorage = require("@chartiq/finsemble").models.baseStorage;
+const Logger = require("@chartiq/finsemble").Clients.Logger;
+
+Logger.start();
+
+class MongoStorageAdapter extends BaseStorage {
+	constructor(uuid) {
+		super(arguments);
+		this.baseURL = "http://localhost:3001";
+
+        this.ROUTES = {
+           	SAVE: `${this.baseURL}/save`,
+           	GET: `${this.baseURL}/get`,
+           	KEYS: `${this.baseURL}/keys`,
+           	DELETE: `${this.baseURL}/delete`
+        }
+	}
+
+	/**
+	 * Get method.
+	 * @param {object} params
+	 * @param {string} params.topic A topic under which the data should be stored.
+	 * @param {string} params.key The key whose value is being set.
+	 * @param {function} cb Callback to be invoked upon completion.
+	 */
+	get = function(params, cb) {
+		
+	}
+
+	/**
+	 * Save method.
+	 * @param {object} params
+	 * @param {string} params.topic A topic under which the data should be stored.
+	 * @param {string} params.key The key whose value is being set.
+	 * @param {any} params.value The value being saved.
+	 * @param {function} cb Callback to be invoked upon save completion.
+	 */
+	save = function(params, cb) {
+		
+	}
+
+    /**
+	 * Returns all keys stored in MongoDB.
+	 *
+	 * @param {object} params
+	 * @param {function} cb
+	 */
+	keys = function(params, cb) {
+
+	}
+
+	/**
+	 * Delete method.
+	 * @param {object} params
+	 * @param {string} params.topic A topic under which the data should be stored.
+	 * @param {string} params.key The key whose value is being deleted.
+	 * @param {function} cb Callback to be invoked upon completion.
+	 */
+	delete = function(params, cb) {
+		
+	}
+}
+
+let mongoAdapter = new MongoStorageAdapter("MongoStorageAdapter")
+module.exports = mongoAdapter; 
+
+
+```
+
+Because keys are a multipart string, it's useful to have a helper function to construct what we'll refer to as the _combined key_. The helper `getCombinedKey` is provided for you in the baseStorage model within Finsemble; by extending our adapter from BaseStorage, it will inherit this helper. You should also include the Logger client for debugging purposes.
```

Comment:
```suggestion
The Storage service in Finsemble provides multiple variables that can be used to address stored data, however, in our MongoDB example, we're going to work with simple key:value pairs. Hence, keys are a multipart strings and therefore it's useful to have a helper function to construct what we'll refer to as the _combined key_. The helper `getCombinedKey` is provided for you in the baseStorage model within Finsemble; by extending our adapter from BaseStorage, it will inherit this helper. You should also include the Logger client for debugging purposes.

Note that you do not have to use the `getCombinedKey` helper, particularly if you intend to store data in a multi-column format using all of the variables provided.
```